pkg/utils/parse_arf_result_test.go

```diff
@@ -302,6 +305,61 @@ Server 3.fedora.pool.ntp.org`
 		})
 
 	})
+
+	Describe("Test for manual Rules", func() {
+		BeforeEach(func() {
+			mcInstance := &mcfgv1.MachineConfig{}
+			schema = scheme.Scheme
+			schema.AddKnownTypes(mcfgv1.SchemeGroupVersion, mcInstance)
+			resultsFilename = "../../tests/data/xccdf-result-remdiation-templating.xml"
+			dsFilename = "../../tests/data/ds-input-for-remediation-value.xml"
+		})
+
+		JustBeforeEach(func() {
+			xccdf, err = os.Open(resultsFilename)
+			Expect(err).NotTo(HaveOccurred())
+
+			ds, err = os.Open(dsFilename)
+			Expect(err).NotTo(HaveOccurred())
+			dsDom, err := ParseContent(ds)
+			Expect(err).NotTo(HaveOccurred())
+			manualRules := []string{}
+			manualRules = append(manualRules, "rhcos4-auditd-data-retention-space-left")
+			resultList, err = ParseResultsFromContentAndXccdf(schema, "testScan", "testNamespace", dsDom, xccdf, manualRules)
+			Expect(resultList).NotTo(BeEmpty())
+			nChecks, nRems = countResultItems(resultList)
```

Comment:
should nRems be 0 ?

Reply:
this is different here because it was using the past result. I will test the remediation in e2e.